### PR TITLE
Strip HTML tags from text response answers & solutions

### LINKED ISSUE #2130: For text response questions, the solutions included the HTML tags from the rich text editor for matching.
- [ ] Allow student answer to have rich-text but strip html tags when comparing
- [ ] Remove rich text editor from solution UI. 
- [ ] Add migration script to strip HTML tags to existing solutions

### PR DESCRIPTION
**Pull request by jeremyyap**
Fixes #2130 

Also includes some fixes for bugs introduced by #2165 
- Instance admins could not download submissions because there was no `course_user`
- HTML tags and base64 images were not stripped from downloaded answers

### CODE REVIEW

**Comment by allenwq** [comment]
LGTM.

File: spec/features/course/assessment/question/text_response_management_spec.rb [hunk 1]
```diff
@@ -83,13 +83,10 @@
           find('a.add_fields', text: link).trigger('click')
           within all('.edit_question_text_response '\
             'tr.question_text_response_solution')[i] do
-            # A custom css selector, :last is added here because +fill_in_rails_summernote+ doesn't
-            # acknowledge the scope defined by capabara.
-            # This works only if +click_link+ is executed before each option.
-            fill_in_rails_summernote '.question_text_response_solutions_solution:last',
-                                     solution[:solution]
+            find('input.text-response-solution').set solution[:solution]
```
Comment (allenwq) [inline spec/features/course/assessment/question/text_response_management_spec.rb]: @jeremyyap why `fill_in_rails_summernote` is removed here ?

Comment (jeremyyap) [inline spec/features/course/assessment/question/text_response_management_spec.rb]: Summernote editor was removed for text response solutions, so can just use set()

Comment (allenwq) [inline spec/features/course/assessment/question/text_response_management_spec.rb]: 👍

Comment (weiqingtoh) [inline spec/features/course/assessment/question/text_response_management_spec.rb]: Might have been better to leave the comment, but it's a minor issue.

Comment (jeremyyap) [inline spec/features/course/assessment/question/text_response_management_spec.rb]: Oh no, I think I accidentally left it out when resolving the conflict

**Pull request merged**